app/containers/message/styles.ts

```diff
@@ -168,5 +170,19 @@ export default StyleSheet.create({
 	threadDetails: {
 		flex: 1,
 		marginLeft: 12
+	},
+	imageBlur: {
+		position: 'absolute',
+		borderWidth: 0,
```

Comment:
Do we need this?

Reply:
yeap, otherwise, it's showing the borderWidth as hairline and color black